# name: "try { ... } catch (Exception $e) { ... }"
# key: try
# --
try {
	$0
} catch (${1:Exception} $e) {
}